### ISSUE TITLE
Quarkus 2.13.5: quarkus-opentelemetry doesn't send data to collector

### ISSUE THREAD

**Issue by enrico-niccoli** [quarkusio/quarkus#30711]
### Describe the bug

Hi everyone,

I'm using Quarkus 2.13.5 with quarkus-opentelemetry to send trace data to an OpenTelemetry collector and then to Jaeger, but my application isn't shown in JaegerUI as probably no data is sent to it.  
This happens only using Quarkus 2.13.5: if I try using Quarkus 2.15.x with the same pom and same application.properties configuration, everything works without any problem.

Is there a minimum Quarkus version to use quarkus-opentelemetry? The guide doesn't say anything about that...

### Expected behavior

My application is shown in JaegerUI and data is correctly sent and displayed

### Actual behavior

My application isn't shown in JaegerUI

### How to Reproduce?

Create a minimal application with Quarkus 2.13.5 and quarkus-opentelemetry dependency. Set application.properties like:
```
quarkus.application.name=test
quarkus.opentelemetry.enabled=true 
quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317
quarkus.log.console.format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId}, sampled=%X{sampled} [%c{2.}] (%t) %s%e%n  

```

Start Jaeger and OpenTelemetry collector through docker, with OTEL receiver on port 4317. I've attached a zip that contains a docker-compose file and a otel configuration file. 
[docker-compose.zip](https://github.com/quarkusio/quarkus/files/10536435/docker-compose.zip)

### Output of `uname -a` or `ver`

Linux asset-00287 5.15.0-58-generic #64-Ubuntu SMP Thu Jan 5 11:43:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

openjdk version "17.0.5"

### Quarkus version or git rev

2.13.5

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

3.10.1

**Comment by brunobat** [comment]
I'm taking a look at it

**Comment by brunobat** [comment]
You need the old opentelemetry-exporter-otlp dependency instead of just the quarkus-opentelemetry one.
opentelemetry-exporter-otlp was eliminated in the meantime and in recent versions is bundled together with  quarkus-opentelemetry.
Does that work for you?

**Comment by enrico-niccoli** [comment]
Yes. Thanks a lot for the explanation, I didn't notice the difference!

**Comment by brunobat** [comment]
Will close the issue
